simulation/__main__.py

```diff
@@ -66,15 +70,16 @@ def float_between_0_and_1(value: str):
 
     parser.add_argument(
         "--intervention",
-        choices=["rhi", "boiler_upgrade_scheme"],
-        type=str,
+        action="append",
+        help="Valid interventions are: rhi, boiler_upgrade_scheme.",
```

Comment:
You could do this so we don't have to update it every time we add an intervention.

```suggestion
        help="Valid interventions are: " + ', '.join(member.name for member in InterventionType)
'RHI, BOILER_UPGRADE_SCHEME',
```